sdk/loadtestservice/azure-developer-loadtesting/README.md

```diff
@@ -0,0 +1,208 @@
+
+`00000000-0000-0000-0000-000000000000.aaa.cnt-prod.loadtesting.azure.com`
+
+The first GUID `00000000-0000-0000-0000-000000000000` is the unique identifier used for accessing the Azure Load Testing resource. This is followed by  `aaa` which is the Azure region of the resource.
+
+The data-plane endpoint is obtained from Control Plane APIs.
+
+**Example:** `1234abcd-12ab-12ab-12ab-123456abcdef.eus.cnt-prod.loadtesting.azure.com`
+
+In the above example, `eus` represents the Azure region `East US`.
+
+## Examples
+
+### Creating a Load Test
+
+```java java-readme-sample-createTest
+LoadTestingClient client = new LoadTestingClientBuilder()
+    .credential(new DefaultAzureCredentialBuilder().build())
+    .endpoint("<endpoint>")
+    .buildClient();
+
+Map<String, Object> testMap = new HashMap<String, Object>();
+testMap.put("displayName", "Sample Display Name");
+testMap.put("description", "Sample Description");
+
+Map<String, Object> loadTestConfigMap = new HashMap<String, Object>();
+loadTestConfigMap.put("engineInstances", 1);
+testMap.put("loadTestConfig", loadTestConfigMap);
+
+Map<String, Object> envVarMap = new HashMap<String, Object>();
+envVarMap.put("a", "b");
+envVarMap.put("x", "y");
+testMap.put("environmentVariables", envVarMap);
+
+BinaryData test = BinaryData.fromObject(testMap);
+
+Response<BinaryData> testOutResponse = client.getLoadTestAdministrationClient().createOrUpdateTestWithResponse("test12345", test, null);
+System.out.println(testOutResponse.getValue().toString());
+```
+
+### Uploading .jmx file to a Load Test
+
+```java java-readme-sample-uploadTestFile
+LoadTestingClient client = new LoadTestingClientBuilder()
+    .credential(new DefaultAzureCredentialBuilder().build())
+    .endpoint("<endpoint>")
+    .buildClient();
+
+BinaryData fileData = BinaryData.fromFile(new File("path/to/file").toPath());
+Response<BinaryData> fileUrlOut = client.getLoadTestAdministrationClient().uploadTestFileWithResponse("test12345", "file12345", "sample-file.jmx", fileData, null);
+System.out.println(fileUrlOut.getValue().toString());
+```
+
+### Running a Load Test
+
+```java java-readme-sample-runTest
+LoadTestingClient client = new LoadTestingClientBuilder()
+    .credential(new DefaultAzureCredentialBuilder().build())
+    .endpoint("<endpoint>")
+    .buildClient();
+
+Map<String, Object> testRunMap = new HashMap<String, Object>();
+testRunMap.put("testId", "test12345");
+testRunMap.put("displayName", "SDK-Created-TestRun");
+
+Map<String, Object> loadTestConfigMap = new HashMap<String, Object>();
+loadTestConfigMap.put("engineInstances", 2);
+testRunMap.put("loadTestConfig", loadTestConfigMap);
+
+BinaryData testRun = BinaryData.fromObject(testRunMap);
+
+Response<BinaryData> testRunOut = client.getLoadTestRunClient().createAndUpdateTestRunWithResponse("testrun12345", testRun, null);
+System.out.println(testRunOut.getValue().toString());
+```
+
```

Comment:
Can we add a  sample code to fetch the client metrics for this test run to complete the end-to-end scenario?